src/core/AnimatedTransform.js

```diff
@@ -23,6 +24,21 @@ function sanitizeTransform(inputTransform) {
   return outputTransform;
 }
 
+function mapMatrixArrays(inputTransform) {
+  return inputTransform.map(transform => {
+    const obj = {};
+    for (const key in transform) {
+      const value = transform[key];
+
+      obj[key] =
+        key === 'matrix' && Array.isArray(value)
+          ? animatedArrayFromNode(value)
```

Comment:
Would it be possible to follow the same methodology for memoizing nodes as we do for props, styles etc. In these cases we memoize nodes using data from `oldNode` object (see `createOrReuse*` methods). Here instead you make a global variable responsible for storing memoized values. This is not ideal as it will leak memory (perhaps not significant leak but still)